registry/storage/tagstore_test.go

```diff
@@ -98,8 +98,8 @@ func TestTagStoreUnTag(t *testing.T) {
 	desc := distribution.Descriptor{Digest: "sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb"}
 
 	err := tags.Untag(ctx, "latest")
-	if err != nil {
-		t.Error(err)
+	if err == nil {
+		t.Error("expected error removing unknown tag")
```

Comment:
The `latest` tag is only created in line 105. So this should be a "try to delete unknown tag" test. This was not a problem so far because we were silencing "not found" errors in `registry/storage/tagstore.go`.